src/task/cluster/ins/v2.rs

```diff
@@ -0,0 +1,230 @@
+use indexmap::IndexMap;
+use serde::{de::Error, Deserialize, Serialize};
+use serde_yaml::Value;
+use std::cmp::Ordering;
+
+use crate::task::cluster::ins::{
+    count_one, default_weight, is_false, is_zero, v1::Instance, IntoV2, Role, Type,
+};
+
+#[derive(Serialize, Clone, Debug, Default, PartialEq, Eq)]
+/// Some tarantool cartridge instance
+///
+/// ```yaml
+/// - name: "catalogue"
+///   type: "storage"
+///   count: 1
```

Comment:
it does not seem clear. is it genin or tarantool format? 
I suppose that V2 configuration should be explained in README, because currently you need to search through whole project (tests, comments) to find examples.